.storybook/preview.js

```diff
@@ -18,6 +18,12 @@ export const parameters = {
       date: /Date$/,
     },
   },
+  options: {
+    storySort: {
+      method: 'alphabetical',
+      order: ['Claim Tracker', 'Component', ['Atom', 'Page Section'], 'Example'],
+    }
```

Comment:
Sets the sidebar to sort first by the specified order, with a fallback to alphabetical.